indico/web/flask/app.py

```diff
@@ -239,6 +242,36 @@ def setup_assets():
     register_all_css(core_env)
 
 
+def _get_webpack_config(app):
+    static_url_path = os.path.join(app.config['APPLICATION_ROOT'] or '/', app.static_url_path)
+    return {
+        'build': {
+            'debug': app.debug,
+            'rootPath': app.root_path,
+            'staticPath': app.static_folder,
+            'staticURL': static_url_path,
+            'distPath': app.config['WEBPACKEXT_PROJECT_DISTDIR'],
+            'distURL': os.path.join(static_url_path, 'dist/')
+        },
+        'themes': theme_settings.themes
+    }
+
+
+def setup_webpack(app):
+    pkg_path = os.path.dirname(get_root_path('indico'))
+    project = WebpackBundleProject(
+        pkg_path,
+        config=lambda: _get_webpack_config(app),
+        config_path=os.path.join(pkg_path, 'config.json')
+    )
+    app.config.update({
+        'WEBPACKEXT_PROJECT': project,
+        'WEBPACKEXT_MANIFEST_PATH': os.path.join('dist', 'manifest.json')
+    })
+
+    FlaskWebpackExt(app)
```

Comment:
see my comment somewhere above about making this global and just calling `init_app` here